app/views/sessions/new.html.haml

```diff
@@ -0,0 +1,12 @@
+
+%section.sign_in.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = bootstrap_form_for @user,url: sessions_path, html: {class: "form-horizontal"} do |f|
+        %header
+          %h1 Sign in
+        %fieldset
+          = f.email_field :email, label: "Email Address"
+          = f.password_field :password
+        %fieldset.form-group.action
+          %input(type="submit" value="Sign in" class="btn btn-default")
```

Comment:
I would prefer this to be `f.submit ...` just to be consistent and more readable.